examples/Makefile

```diff
@@ -189,13 +189,15 @@ deploy-prog: kustomize
 ifndef TAG
 	sed 's@URL_BC@$(IMAGE_BC)@' config/$(CONFIG_DIR)/patch.yaml.env > config/$(CONFIG_DIR)/patch.yaml
 	cd config/$(CONFIG_DIR) && $(KUSTOMIZE) edit set image quay.io/bpfman-userspace/$(PROG_NAME)=${IMAGE_US}
+	$(eval KUST_DIR=$(CONFIG_DIR))
 else
-	$(eval KUST_DIR=$(TAG))
```

Comment:
@anfredette This look OK to you? KUST_DIR was being set but not used. So loading from a release, like `TAG=v0.4.1 make deploy` was just loading latest.